eip/EIPS/eip-4337.md

```diff
@@ -482,44 +487,44 @@ Paymasters facilitate transaction sponsorship, allowing third-party-designed mec
+It is an important design goal of this proposal to replicate the key property of EOAs that users do not need to perform some custom action or rely on an existing user to create their account; they can simply generate an address locally and immediately start accepting funds.
 
-The wallet creation itself is done by a "factory" contract, with wallet-specific data.
-The factory is expected to use CREATE2 (not CREATE) to create the wallet, so that the order of creation of wallets doesn't interfere with the generated addresses.
+The account creation itself is done by a "factory" contract, with account-specific data.
+The factory is expected to use CREATE2 (not CREATE) to create the account, so that the order of creation of accounts doesn't interfere with the generated addresses.
 The `initCode` field (if non-zero length) is parsed as a 20-byte address, followed by "calldata" to pass to this address.
-This method call is expected to create a wallet and return its address.
-If the factory does use CREATE2 or some other deterministic method to create the wallet, it's expected to return the wallet address even if the wallet has already been created.  This is to make it easier for clients to query the address without knowing if the wallet has already been deployed, by simulating a call to `entryPoint.getSenderAddress()`, which calls the factory under the hood.
+This method call is expected to create an account and return its address.
+If the factory does use CREATE2 or some other deterministic method to create the account, it's expected to return the account address even if the account has already been created.  This is to make it easier for clients to query the address without knowing if the account has already been deployed, by simulating a call to `entryPoint.getSenderAddress()`, which calls the factory under the hood.
 When `initCode` is specified, if either the `sender` address points to an existing contract, or (after calling the initCode) the `sender` address still does not exist,
 then the operation is aborted.
 The `initCode` MUST NOT be called directly from the entryPoint, but from another address.
 The contract created by this factory method should accept a call to `validateUserOp` to validate the UserOp's signature.
 For security reasons, it is important that the generated contract address will depend on the initial signature.
-This way, even if someone can create a wallet at that address, he can't set different credentials to control it.
+This way, even if someone can create an account at that address, he can't set different credentials to control it.
 
-NOTE: In order for the wallet to determine the "counterfactual" address of the wallet (prior its creation), 
+NOTE: In order for the account to determine the "counterfactual" address of the account (prior its creation), 
```

Comment:
Only change the 2nd "wallet" to "account"  It's the *wallet* determining the "counterfactual" address of the *account*.